package.json

```diff
@@ -11,6 +11,8 @@
     "clean:media": "docker-compose exec media rm -rf ./.media",
     "lint": "nx run-many --target=lint",
     "build": "nx run-many --targets=build",
+    "docker:build:api": "npx nx run-many --targets=build --projects=api-example",
```

Comment:
Imo don't think these are needed and we could just run `npx nx build api-example` in the dockerfile. I don't like adding simple commands to the package.json. What do others think?

There's a few other scripts I would get rid of too :D They are mostly there bc of historic reasons (yarn time and workspaces)